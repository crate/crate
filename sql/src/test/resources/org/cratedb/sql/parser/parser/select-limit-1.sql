SELECT * FROM t1 FETCH FIRST 10 ROWS ONLY
