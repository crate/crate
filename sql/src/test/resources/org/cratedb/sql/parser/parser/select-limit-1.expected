org.cratedb.sql.parser.parser.CursorNode@2c006765
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@4e842e74
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@4805e9f1

		[0]:		
		org.cratedb.sql.parser.parser.AllResultColumn@7c41f227
		tableName: null
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@4d811e2c

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@57e40274
		tableName: t1
		updateOrDelete: null
		null
		correlation Name: null
		null
fetchFirst: 	
	org.cratedb.sql.parser.parser.NumericConstantNode@3a5d3ac0
	value: 10
	type: INTEGER NOT NULL